deploy/deploy.lib

```diff
@@ -653,8 +653,5 @@ EOF
     fi
 
     cp "${HOME}/.spack/compilers.yaml" "${HOME}" || true
-
-    log "cleaning up after a bad spack"
-    spack clean -a
```

Comment:
Do we want to keep this in here? Since the logs should show up in the XML, I would like to keep the deployment directories as clean as possible.

Reply:
restored